releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,38 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 2.6.0-preview - April 17, 2023
+
+### Feature - RequirementType
+
+Features can now declare a `RequirementType`. The default `RequirementType` is `Any`, which means if any of it's filters evaluate to true, then the feature will be enabled. Declaring a `RequirementType` of `All` means that every filter must evaluate to true in order for the feature to be enabled. 
```

Comment:
Could you also add a link to your PRs in the Feature Management repo? 
For example, you could add [#221](https://github.com/microsoft/FeatureManagement-Dotnet/pull/221) at the end of this sentence.